### DISCUSSION TITLE
Node Red Export funktioniert nicht

### DISCUSSION THREAD

**Discussion by ToniBecause** [camueller/SmartApplianceEnabler#462] [Q&A]
Hallo, ich bin grade etwas verwirrt, ich will lediglich ein Export meiner So Zähler für Node red machen um diese letztendlich in Home Assistant sehen zu können. Aber wenn ich auf den Button klicke zum Export wird das Browserfenster dunkler aber sonst passiert nichts.
Das Node red dashboard läuft (ohne daten) und auch in die node red oberfläche komme ich rein.
Die 2 Paletten habe ich auch schon installiert und camueller/node-red-contrib-ui-timelines-chart sowie date-fns auch. wobei ich glaube hier den falschen User genommen zu haben. Das ist leider nicht ganz eindeutig in der Anleitung. Eventuell habe ich es dann ins falsche Verzeichnis installiert? Wie komme ich denn ggf. in das richtige Verzeichnis falls es daran liegen könnte?
Jemand ein Tip wo ich mal suchen könnte? Leider kenne ich mich mit den Kommandobefehlen nicht so gut aus.
Ich nutze das Terminal vom Webmin und Version 2.2.0 vom SAE.

**Comment by camueller** [comment]
Habe es gerade mal (weider) getestet: Tatsächlich erscheint das Fenster mit den Export-Daten nicht. Das muss ich mir mal anschauen ...

**Comment by ToniBecause** [comment]
OK danke, dann besteht ja noch Hoffnung für mich :-D

**Comment by camueller** [comment]
Danke für den Hinweis. Ich habe den Fehler schon gefixt und baue noch einen Test dafür, damit das nicht wieder kaputtgehen kann. In den nächstn Tagen werde ich eine Bugfix-Version dafür releasen.

**Reply by ToniBecause** [reply]
Wow, super. Dankeschön. Ich werde das Bugfix dann sofort installieren.

**Comment by camueller** [comment]
2.2.1 ist releast.

**Reply by ToniBecause** [reply]
Super, Dankeschön. Habe es schon installiert und ein Export scheint zu klappen. Inhaltlich muss ich dann heute Abend mal in Ruhe ausprobieren ;-)
